hooks/002.1-add-gdm.chroot

```diff
@@ -28,33 +28,9 @@ apt-get install --no-install-recommends -y \
     gir1.2-gsound-1.0 \
     libcanberra-pulse \
     libsnapd-glib-2-1 \
-    ibus
-
-# Install a basic Ubuntu session
-apt-get install --no-install-recommends -y \
-    ubuntu-session \
-    gkbd-capplet \
-    gnome-menus \
-    gnome-terminal \
-    gnome-settings-daemon \
-    gnome-shell-extension-appindicator \
-    gnome-shell-extension-desktop-icons-ng \
-    gnome-shell-extension-ubuntu-dock \
-    gnome-shell-extension-ubuntu-tiling-assistant \
-    gnome-control-center \
-    gnome-keyring \
-    fprintd \
-    libpam-fprintd \
-    opensc \
-    bolt \
-    libpam-gnome-keyring \
-    spice-vdagent \
-    xdg-desktop-portal \
-    xdg-desktop-portal-gnome \
-    xdg-desktop-portal-gtk \
-    xdg-user-dirs-gtk \
-    inotify-tools \
-    alsa-ucm-conf
+    ibus \
+    pkexec \
+    xdg-desktop-portal
 
 # Remove setuid from some executables we're not using
 chmod u-s,g-s /usr/bin/pkexec
```

Comment:
Maybe you needed to add pkexec to make this bit of the hook pass?

Reply:
Yes, that was the reason O:-) Removed too.